proc_dash/app.py

```diff
@@ -183,7 +183,8 @@ def process_bagel(contents, filename):
         data, total_subjects, sessions, upload_error = util.parse_csv_contents(
             contents=contents, filename=filename
         )
-    except Exception:
+    except Exception as exc:
+        print(exc)  # for debugging
```

Comment:
You can take a look at https://docs.python.org/3/library/logging.html to collect these types of messages and handle them - it gives you more clean control over what the output and where

Reply:
(for a later PR)